django/contrib/auth/views.py

```diff
@@ -175,6 +175,7 @@ def password_reset(request, is_admin_site=False,
                    post_reset_redirect=None,
                    from_email=None,
                    extra_context=None,
+                   extra_email_context=None,
```

Comment:
To make this backwards compatible with older Django versions, `extra_email_context=None` should be the last item.